AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -6900,6 +6899,8 @@ kraj.by#$#div[aria-labelledby="CModalLabel"] ~ div.modal-backdrop { display: non
 !+ NOT_OPTIMIZED
 kraj.by#$#body { overflow: auto !important; padding-right: 0 !important; }
 !+ NOT_OPTIMIZED
+dailytrust.com##main + div[style*="position: fixed; left: 0px; top: 0px; width: 100%; height: 100%;"]
```

Comment:
Strange. Have you compared these files? I see the rule in the optimized filter.
https://filters.adtidy.org/extension/safari/filters/14.txt
https://filters.adtidy.org/extension/safari/filters/14_optimized.txt
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/149534278-08adbded-9e7d-4c28-8cbd-eab1cdec417c.png)
</details><br/>

Reply:
If it does not work, better ask in our AG for Safari chat.